doc/usage/extensions/napoleon.rst

```diff
@@ -550,9 +575,10 @@ sure that "sphinx.ext.napoleon" is enabled in ``conf.py``::
    .. versionadded:: 3.4
 
 .. confval:: napoleon_custom_sections
+   :type: `Optional[Sequence[str | Sequence[str, str]]]`
```

Comment:
Is it necessary to use `Sequence` instead of `list` here? And should I use the `|`, or nested `Optional`?